### PR TITLE
PR: Help People Get Started Faster!

### PR DESCRIPTION
**Pull request by nelsonic**
Addresses: https://github.com/dwyl/learn-flutter/issues/75

### CODE REVIEW

**Review by Arhell — DISMISSED** [review]

**Comment by nelsonic** [comment]
@SimonLab please take a quick look at these docs additions and merge if ok. 🙏

File: README.md [hunk 1]
```diff
@@ -384,40 +449,81 @@ run the `flutter` command, this should prop up.
...
+Change into the `demo_app` directory:
+
+```sh
+cd learn-flutter/demo_app
+```
+
+Install the necessary dependencies:
+
+```sh
+
```
Comment (SimonLab) [inline README.md:482]:
```suggestion
```sh
flutter pub get
```

File: README.md [hunk 2]
```diff
@@ -630,20 +733,16 @@ void main() {
...
-The `State` object is declared right after, 
-initializing an `int _counter` at `0`.
-It declares an `_increment()` function 
-that calls `setState()` 
-(indicating the state is going to be changed) 
-and increments the `_counter` variable.
-
-As with any widget, 
-the `build()` method makes use of the `_counter` variable
+declares its' state by using the `State` object.
```
Comment (SimonLab) [inline README.md:739]:
```suggestion
declares its state by using the `State` object.
```

File: README.md [hunk 3]
```diff
@@ -264,6 +288,7 @@ and press `Install`. It's that easy.
 <img width="1292" alt="image" src="https://user-images.githubusercontent.com/17494745/200554456-ff9dc9cb-7a2a-4eb8-aff4-90dfd00a0427.png">
 
 ### Install Android Studio
+
 Now targetting for Android devices, we need to install Android SDK and toolkits.
```
Comment (SimonLab) [inline README.md:292]:
```suggestion
Now targeting for Android devices, we need to install Android SDK and toolkits.
```

**Review by SimonLab — APPROVED** [review]
Looks good

**Comment by nelsonic** [comment]
Thanks @SimonLab 👍

**Pull request merged**